### PR TITLE
[rapid-response] Aligned certificate verification enable/disable flag to the other charts

### PR DESCRIPTION
**Pull request by dark-vex**
## What this PR does / why we need it:
Aligned the certificate verification enable/disable flag behaviour to the other charts(`sslVerifyCertificate`)

## Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [X] Chart Version bumped
- [X] Variables are documented in the README.md
- [X] Title of the PR starts with chart name (e.g. `[mychartname]`)
- [ ] PR only contains changes for one chart

### CODE REVIEW

**Comment by dark-vex** [comment]
@achandras @lilx1ao when you have time could you please review it? Thanks!

**Comment by lilx1ao** [comment]
@dark-vex merge master!

File: charts/rapid-response/templates/_helpers.tpl [hunk 1]
```diff
@@ -170,4 +170,18 @@ Determine collector endpoint based on provided region or .Values.rapidResponse.a
...
+{{- end -}}
+
+{{/*
+Rapid Response have the environment variable skip_tls_check: true for skip the certficate verification
+while we do the other way round for our other components (sslVerifyCertificate: false for disabling the check).
+The aim of rapidResponse.certificateValidation is to align the settings with the other Sysdig charts,
+without introducing changes on Rapid Response container image.
+*/}}
+{{- define "rapidResponse.certificateValidation" -}}
+    {{- if or ( eq (.Values.rapidResponse.skipTlsVerifyCertificate | toString) "true") (eq (.Values.rapidResponse.sslVerifyCertificate | toString) "false") -}}
```
Comment (lilx1ao) [inline charts/rapid-response/templates/_helpers.tpl:182]: you don't need toString and compare it with a string.

File: charts/rapid-response/templates/_helpers.tpl [hunk 2]
```diff
@@ -170,4 +170,18 @@ Determine collector endpoint based on provided region or .Values.rapidResponse.a
...
+        {{- "true" -}}
```
Comment (lilx1ao) [inline charts/rapid-response/templates/_helpers.tpl:183]: this doesn't need to be a string

File: charts/rapid-response/templates/_helpers.tpl (hunk 1 above)
Comment (dark-vex) [reply charts/rapid-response/templates/_helpers.tpl:182]: So you mean creating an if like that:
```
{{- if or (.Values.rapidResponse.skipTlsVerifyCertificate) (.Values.rapidResponse.sslVerifyCertificate) -}}
```
but how I can specify `Values.rapidResponse.sslVerifyCertificate` as false?

Comment (lilx1ao) [reply charts/rapid-response/templates/_helpers.tpl:182]: not Values.rapidResponse.sslVerifyCertificate

**Pull request merged**